### PR TITLE
deprecate(base): FairModule::getFairVolume

### PR DESCRIPTION
**Pull request by ChristianTackeGSI**
I could not find any caller anywhere.

---

Checklist:

* [X] Followed the [Contributing Guidelines](https://github.com/FairRootGroup/FairRoot/blob/dev/CONTRIBUTING.md)

## Summary by CodeRabbit

- **Refactor**
	- Deprecated the `getFairVolume` method in the `FairModule` class, scheduled for removal in version 20.

### CODE REVIEW

File: fairroot/base/sim/FairModule.h
Comment (dennisklein) [inline fairroot/base/sim/FairModule.h:147]: Hm, are we sure we can hold this promise? How about being a bit less specific via "will be removed in a future release" or similar?

Second thought, not sure, if `chore` and deprecation fit. What's your definition for chore?

Comment (ChristianTackeGSI) [reply fairroot/base/sim/FairModule.h:147]: I don't know when / how we can hold those promisses. I just wanted to schedule it somehow.  (Otherwise we will keep it for the next 24 years like `TList(TObject*)` :wink: .)

Using `\deprecated` puts it at least on the deprecation list in Doxygen which we could work on and actually remove things.

Maybe we should decide on our deprecation strategy at some point?

Comment (ChristianTackeGSI) [reply fairroot/base/sim/FairModule.h:147]: My defintion of "chore" is mostly like "clean things up, or prepare to clean up".

That said: I am totally open to alternative tags for deprecations?

Comment (dennisklein) [reply fairroot/base/sim/FairModule.h:147]: how about `deprecate(base): FairModule::getFairVolume`?

Comment (karabowi) [reply fairroot/base/sim/FairModule.h:147]: I would vote for the `deprecate(base): FairModule::getFairVolume` comment.

Comment (ChristianTackeGSI) [reply fairroot/base/sim/FairModule.h:147]: done.

looks good to me!

**Review by karabowi — APPROVED** [review]

**Pull request merged**
